clerk_test.go

```diff
@@ -385,3 +412,38 @@ func TestBackendCall_NonParseableError(t *testing.T) {
 	// familiar API error response.
 	assert.Equal(t, errorResponse, err.Error())
 }
+
+// TestBackendCall_Multipart tests multipart/form-data requests.
+func TestBackendCall_Multipart(t *testing.T) {
+	ctx := context.Background()
+	name := "the-name"
+
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// The content type will be multipart/form-data
+		assert.Contains(t, r.Header.Get("Content-Type"), "multipart/form-data")
+		// The body contains the "name" field
+		err := r.ParseMultipartForm(1024)
+		require.NoError(t, err)
+		assert.Equal(t, name, r.Form.Get("name"))
+
+		_, err = w.Write([]byte(`{}`))
+		require.NoError(t, err)
+	}))
+	defer ts.Close()
+
+	// Set up a mock backend which triggers requests to our test server above.
+	SetBackend(NewBackend(&BackendConfig{
+		HTTPClient: ts.Client(),
+		URL:        &ts.URL,
```

Comment:
🙃 I just noticed this now: should this field be names `URL` or `BaseURL` 🤔 ?

Reply:
I think I like `URL`.